src/betterproto/plugin/models.py

```diff
@@ -503,16 +504,13 @@ def __post_init__(self):
                     self.proto_v_type = self.proto_obj.Type.Name(nested.field[1].type)
         super().__post_init__()  # call FieldCompiler-> MessageCompiler __post_init__
 
-    def get_field_string(self, indent: int = 4) -> str:
-        """Construct string representation of this field."""
-        name = f"{self.py_name}"
-        annotations = f": {self.annotation}"
-        betterproto_field_type = (
-            f"betterproto.map_field("
-            f"{self.proto_obj.number}, betterproto.{self.proto_k_type}, "
-            f"betterproto.{self.proto_v_type})"
-        )
-        return name + annotations + " = " + betterproto_field_type
+    @property
+    def betterproto_field_args(self) -> str:
+        return f", betterproto.{self.proto_k_type}, betterproto.{self.proto_v_type}"
```

Comment:
As a side note (for another PR maybe) I think this should return a list that `get_field_string` uses in `",".join(..)`.

Reply:
Implemented, worked great